.babelrc

```diff
@@ -16,14 +16,16 @@
       "displayName": true
     }],
     "inline-react-svg",
+    "lodash",
+    "loadable-components/babel",
     ["module-resolver", {
       "extensions": [".js", ".jsx", ".ts", ".tsx"],
       "root": ["./src"],
       "alias": {
         "reaction": "@artsy/reaction/dist"
       }
     }],
-    "relay"
+    ["relay", { "artifactDirectory": "./src/__generated__", "extensions": ".js,.ts" }]
```

Comment:
For what reason do we need to support `.js` files here?

Reply:
Ah I see, this all is currently done in `.js` files. I assume you’ll be turning it into TS before this PR is done? I’d then also like to remove the `.js` here, because otherwise we may make it too easy for people to take the wrong shortcut.

Reply:
@alloy - yup, was done in js just to sketch, but was going to be polished in TS.